factorBBSize: 1.0
GaussianBlurSize: 3
MaximumFilterSize: 10
